src/components/TwitchSidebar/Streams.ts

```diff
@@ -0,0 +1,60 @@
+const streams = {
```

Comment:
remove all of this, we wont be storing any of the info. The only thing we need to store is an array of ids on the cookbook object